macros/base/end_print.cfg

```diff
@@ -45,11 +46,11 @@ gcode:
     {% endif %}
 
 
-    # If a filter is connected, filter the air at full power
+    # If a filter is connected, and used during the print, continue filtering the air
     # for a couple of min before stopping everything
-    {% if filter_enabled %}
+    {% if filter_enabled and filter_speed > 0 %}
         {% set FILTER_TIME = params.FILTER_TIME|default(filter_default_time)|int %}
-        START_FILTER SPEED=1
+        START_FILTER SPEED={filter_speed / 100}
```

Comment:
Can you keep the value of "1" here?
I prefer it to go at maximum speed at the end of the print to filter a maximum of air before opening the doors.

Reply:
Sure, makes sense.